Umount.sh

```diff
@@ -6,5 +6,6 @@
 
 for BLK in $(mount | grep "${CHROOT}" | awk '{ print $3 }' | sort -r)
 do
-   umount "${BLK}"
+   # umount everything not mounted via fstab
+   cat /etc/fstab | cut -f2 | grep "^${BLK}$" || umount "${BLK}"
```

Comment:
This is going to `cat /etc/fstab` for every mount. Can we account for this in the `for` iterator, instead?

Reply:
Reasons:
- 'mount -a' creates a lot of failures from the running OS.
- 'mount -a' could accidently umount filesystems from the running OS

Reply:
I replaced the Commit with a clean solution.
There was a big issue in `Umount.sh:`
`grep "${CHROOT}" `
CHROOT is empty in this situation and all entries from 'mount' are passed to the for loop...

Reply:
@Rendanic
Damn, son: where are you even encountering this? Given the typical build-methods (encapsulated in spel - where the `umount` is mostly pointless given the pre-registration shutdown of the build-node – or as an interactive, "build to a secondary disk" - where the CHROOT val is set in the env), surprised you ran into issues (i.e., "why I was too lazy to bother with it, previously")

Reply:
I did a lot of tests and used the Umount.sh for umount without stopping my build machine. Otherwise the creation of the Snapshot + AMI isn't 100% safe due to mounted filesystems.

I pushed a new commit with the correct definition of CHROOT - hopefully this is the solution for this PR.